Kernel/Bus/PCI/DeviceTreeInitializer.cpp

```diff
@@ -243,9 +242,30 @@ void initialize()
                 auto const* interrupt_controller = device_tree.phandle(interrupt_controller_phandle);
                 VERIFY(interrupt_controller);
 
+                if (!interrupt_controller->has_property("interrupt-controller"sv)) {
```

Comment:
To be completely correct, you should do this check on the domain root.

Reply:
Well `interrupt_controller` should be set to the domain root. So it's not really your fault. You can just keep it as is if you don't feel like fixing it.